README.md

```diff
@@ -120,9 +120,9 @@ Data which is stored upon startup includes:
 
 * It is easiest to use the development virtual machine to work on this project. Follow [these instructions](RemoteDesktopToVM.md) to set up and connect to the virtual machine using your Gordon account.
 
-* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash.
+* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash. It is possible that you will need to [add the SSH key to your Git account](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh). You will then need to `cd ssh` then `cat id_rsa.pub`, copy the output and go to your github settings and paste it in your SSH keys.
```

Comment:
I think the link describes the process well.  So don't say "You will then need to ...".  Either delete that sentence, or change the transition to something like "It will guide you to ...".

Also `cd ssh` should be `cd .ssh` (notice the dot) on Mac and linux (I don't know about Windows).